feat: Refresh tree shortcut

gytic opened the pull request:
This PR should fix #118 

To refresh the tree view my idea is to:
1. clear existing nodes (under root)
2. Re-initialize nodes (under root)
3. Reset search state
4. Close opened table if it no longer exists

gytic commented:
@jorgerojas26 do you have an idea where and how to close the opened table if it was deleted?

I will look tomorrow a bit more into the code base to figure out how it is opened etc but maybe you have an idea already how one could implement that.

jorgerojas26 commented:
I'm not sure what you mean but maybe you just delete the tab that has the table open.

gytic commented:
Okay I think this is the best I can do.

When refreshing, all nodes under the root node get deleted and afterwards re added.

ccoVeille commented on components/Tree.go:
```diff
@@ -557,3 +528,48 @@ func (tree *Tree) ExpandAll() {
+func (tree *Tree) Refresh(dbName string) {
+	rootNode := tree.GetRoot()
+	rootNode.ClearChildren()
+	// readd nodes
```
```suggestion
	// read nodes
```

Or


```suggestion
	// re-add nodes
```

Using readd is ambiguous

gytic replied:
this is fixed now, thanks for pointing out

gytic commented:
PR is ready for review

jorgerojas26 commented:
This is good enough for now, i think. Can you please just change the keybinding from 'r' to 'R'. The reason is, there is a 'R' keymap for the table that refreshes the data, so, to keep consistency let's use 'R' also for the tree.

With that change, i'm ready to merge.

Thank you very much

gytic commented:
I will work on it this evening

gytic commented:
I ran the lints locally and had no errors. I think this should be now fixed.

Also changed the shortcut from `r` to `R`.
PR should be ready for merge now.
